### PR TITLE
Init model

### PR DESCRIPTION
**Pull request by Dat1606**

### CODE REVIEW

File: app/models/relationship.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Relationship < ApplicationRecord
```
Comment (huongnguyenmta) [inline app/models/relationship.rb]: thêm relation cho nó đi em

**Comment by Dat1606** [comment]
https://edu-redmine.framgia.vn/issues/848

**Comment by Dat1606** [comment]
ready

**Comment by huongnguyenmta** [comment]
merged
[CC][To:1790715] ♍ Tran Xuan Thang 陳春勝  ☯
merge pull init model này ngen em ;)

**Pull request merged**